flutter_vlc_player/example/lib/single_tab.dart

```diff
@@ -75,19 +88,19 @@ class _SingleTabState extends State<SingleTab> {
     fillVideos();
     selectedVideoIndex = 0;
     //
-    var initVideo = listVideos[selectedVideoIndex];
+    final initVideo = listVideos[selectedVideoIndex];
     switch (initVideo.type) {
       case VideoType.network:
         _controller = VlcPlayerController.network(
           initVideo.path,
           hwAcc: HwAcc.full,
           options: VlcPlayerOptions(
             advanced: VlcAdvancedOptions([
-              VlcAdvancedOptions.networkCaching(2000),
+              VlcAdvancedOptions.networkCaching(_networkCaching),
```

Comment:
Add units to name?

Reply:
Did not understand. Do you mean that we need to rename it to `_2000` or this ask was to another variable?

Reply:
I mean the variable name should describe the units.
In this particular case it's milliseconds:
```suggestion
              VlcAdvancedOptions.networkCaching(_networkCachingMs),
```